scripts/kind.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 kind: Cluster
-apiVersion: kind.sigs.k8s.io/v1alpha3
+apiVersion: kind.x-k8s.io/v1alpha4
```

Comment:
@edwarnicke I've tested locally nsm works correctly with this version.

Reply:
@Do you mean kind works correctly with v1alpha4 or v1alpha3?

Reply:
I meant this version `kind.x-k8s.io/v1alpha4`